Source/SteveSloth/Private/MyEnemyAIDeadState.cpp

```diff
@@ -0,0 +1,34 @@
+/****************************************************************************************
+ * Copyright:
+ * Name: Guy Lapensee
+ * Script: ItemBaseClass
+ * Date: April 29, 2024
+ * Description: Enemy AI dead state for enemy base class
+ * TODO:
+ * Known Bugs:
+ ****************************************************************************************/
+
+
+#include "MyEnemyAIDeadState.h"
+
+MyEnemyAIDeadState::MyEnemyAIDeadState()
+{
+	isDead = false;
```

Comment:
This variable name does not conform to the standards in the examples
Should be IsDead with a capital I